images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG PARENT_DEBIAN_FLAVOR
-FROM oryxdevmcr.azurecr.io/public/oryx/build:github-${PARENT_DEBIAN_FLAVOR} AS main
 ARG DEBIAN_FLAVOR
+FROM oryxdevmcr.azurecr.io/public/oryx/cli-${DEBIAN_FLAVOR}:${PARENT_DEBIAN_FLAVOR} AS main
```

Comment:
Just a random question, not a comment on the code: do you know the difference between DEBIAN_FLAVOR and PARENT_DEBIAN_FLAVOR? In which case are these different?

Reply:
Currently, [this](https://github.com/microsoft/Oryx/blob/main/build/buildBuildImages.sh#L232) is the arg for PARENT_DEBIAN_FLAVOR which is essentially just the parent/base image's tag.

Reply:
Thanks for your reply! So just to be sure I got it right, `github` is the value passed as `PARENT_DEBIAN_FLAVOR`? If yes then that seems like something else we'll need to rename as some point.

Reply:
From [here](https://github.com/microsoft/Oryx/blob/main/build/buildBuildImages.sh#L240), the value of PARENT_DEBIAN_FLAVOR is github-actions-debian-bullseye for the azfunc-jamstack-debian-bullseye image.